crates/re_log_types/src/component_types/tensor.rs

```diff
@@ -424,20 +435,33 @@ impl Tensor {
     }
 
     pub fn is_shaped_like_an_image(&self) -> bool {
```

Comment:
we can simplify this quite a bit:

```suggestion
    pub fn is_shaped_like_an_image(&self) -> bool {
        self.image_height_width_channels().is_some()
    }
```